lib/licensed/sources/go.rb

```diff
@@ -160,7 +158,7 @@ def package_info(import_path)
       #
       # args - additional arguments to `go list`, e.g. Go package import path
       def package_info_command(*args)
-        Licensed::Shell.execute("go", "list", "-json", *Array(args)).strip
+        Licensed::Shell.execute("go", "list", "-e", "-json", *Array(args)).strip
```

Comment:
using `-e` in the command sets any errors that are found to a packages `package["Error"]["Err"]` output field rather than exiting non-zero and printing the error to stderr.